### DISCUSSION TITLE
Windows 10 battery and network flyouts do not work

### DISCUSSION THREAD

**Discussion by Dreamescaper** [valinet/ExplorerPatcher#799] [Q&A]
In most cases, clicking on battery or network icon leads to no effect for me, apart from loading indicator showing for a couple of seconds.
Sometimes is _does_ appear, I'm not sure in which cases though.

Windows Build Number: 22000.469
ExplorerPatcher version: 22000.469.41.16

I've found the following entries in EventViewer:
<details><summary>Error</summary>

Faulting application name: ShellExperienceHost.exe, version: 10.0.22000.132, time stamp: 0xdd210a66
Faulting module name: dxgi.dll, version: 22000.469.41.16, time stamp: 0x61f204a2
Exception code: 0xc0000005
Fault offset: 0x00000000000178b1
Faulting process id: 0x125c
Faulting application start time: 0x01d81b739871502c
Faulting application path: C:\Windows\SystemApps\ShellExperienceHost_cw5n1h2txyewy\ShellExperienceHost.exe
Faulting module path: C:\Windows\SystemApps\ShellExperienceHost_cw5n1h2txyewy\dxgi.dll
Report Id: b8fda75b-2358-4e16-967b-33a907915572
Faulting package full name: Microsoft.Windows.ShellExperienceHost_10.0.22000.71_neutral_neutral_cw5n1h2txyewy
Faulting package-relative application ID: App

</details>

<details><summary>Information</summary>

Fault bucket 1244781368081769456, type 5
Event Name: MoAppCrash
Response: Not available
Cab Id: 0

Problem signature:
P1: Microsoft.Windows.ShellExperienceHost_10.0.22000.71_neutral_neutral_cw5n1h2txyewy
P2: praid:App
P3: 10.0.22000.132
P4: dd210a66
P5: dxgi.dll
P6: 22000.469.41.16
P7: 61f204a2
P8: c0000005
P9: 00000000000178b1
P10: 

Attached files:
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.41792745-19ea-4041-a532-c5c3d24a9c3e.tmp.mdmp
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.4603c3a2-fe96-4fda-a4ca-6a5208c3915a.tmp.WERInternalMetadata.xml
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.a0ce03e1-0616-4929-b2a0-d9577217961e.tmp.xml
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.7dbd595c-fed1-4923-9f71-f83bfd312ff2.tmp.csv
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.6fbcfd4d-ab3c-4957-a081-b71ea363b933.tmp.txt

These files may be available here:
\\?\C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppCrash_Microsoft.Window_a631ac7f47b21cc9fded616c6f3a347aeac2de8c_b9d239b3_5d1edeb5-11d3-4b24-bc89-328930baf621

Analysis symbol: 
Rechecking for solution: 0
Report Id: b8fda75b-2358-4e16-967b-33a907915572
Report Status: 268435456
Hashed bucket: e121f9be03a9853351465a0f2389e3f0
Cab Guid: 0
</details>

Some attached file mentioned in Event Viewer: 
[Report.wer](https://github.com/valinet/ExplorerPatcher/files/8010569/Report.wer.txt)

(I've tried to collect full dump based, but failed so far, probably I'm doing something wrong)

**Comment by Dreamescaper** [comment]
Re-checked this issue with latest ExplorerPatcher preview (22000.493.42.9), it's still there, unfortunately :(

**Reply by valinet** [reply]
It’s kind of expected, there was no commit doing anything about it. It happens VERY rarely for me as well, but the VAST majority of times it works just fine. It’s a hack anyway, it’s nice that it works at all, y’know…

**Comment by sdw4938** [comment]
This happens to me. The Wi-Fi flyout works fine but on the other hand the battery flyout crashes 100% of the time

**Reply by valinet** [reply]
Yeah, idk, use Battery Mode app, it’s much better anyway.
